opm/simulators/wells/StandardWell.hpp

```diff
@@ -254,6 +254,25 @@ namespace Opm
                                     DeferredLogger& deferred_logger) const;
 
 
+        std::vector<double> getPrimaryVars() const override
+        {
+            const int num_pri_vars = this->primary_variables_.numWellEq();
+            std::vector<double> retval(num_pri_vars);
+            for (int ii = 0; ii < num_pri_vars; ++ii) {
+                retval[ii] = this->primary_variables_.value(ii);
+            }
+            return retval;
+        }
+
+        int setPrimaryVars(std::vector<double>::const_iterator it) override
```

Comment:
That interface would be a lot better with a [`span<>`](https://en.cppreference.com/w/cpp/container/span) type.  I'm so much looking forward to using C++20.